sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -377,12 +377,15 @@ public void testSetGlobal() throws Exception {
 
     @Test
     public void testSetSession() throws Exception {
-        printStatement("set some_setting = '1'");
-        printStatement("set some_setting = '1', other_setting = 2");
+        printStatement("set global some_setting = '1'");
+        printStatement("set global some_setting = '1', other_setting = 2");
+        printStatement("set some_setting = DEFAULT");
         printStatement("set session some_setting = '1'");
-        printStatement("set session some_setting = '1', other_setting = 2");
+        printStatement("set session some_setting TO DEFAULT");
         printStatement("set session some_setting TO '1'");
-        printStatement("set session some_setting TO '1', other_setting TO 2");
+        printStatement("set local some_setting = '1'");
+        printStatement("set local some_setting TO '1'");
+        printStatement("set local some_setting = DEFAULT");
```

Comment:
test for `SET SESSION some_setting = 1, 2, 3` is missing
[postgres doc](https://www.postgresql.org/docs/8.3/static/sql-set.html):
> New value of parameter. Values can be specified as string constants, identifiers, numbers, or comma-separated lists of these, as appropriate for the particular parameter.